--- MODULE.bazel	2024-08-26 17:43:01.000000000 +0900
+++ MODULE.bazel.new	2024-08-26 22:38:15.450488362 +0900
@@ -148,6 +147,18 @@
     packages = ["glib-2.0", "gobject-2.0", "ibus-1.0"],
 )
 
+# Fcitx
+pkg_config_repository(
+  name = "fcitx",
+  packages = ["fcitx"],
+)
+
+# Fcitx 5
+pkg_config_repository(
+  name = "fcitx5",
+  packages = ["Fcitx5Module", "Fcitx5Core"],
+)
+
 # Qt for Linux
 pkg_config_repository(
     name = "qt_linux",
